api/v1alpha2/helpers.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package v1alpha2
+
+import (
+	tfc "github.com/hashicorp/go-tfe"
+)
+
+func (rs *RunStatus) RunCompleted() bool {
+	return runCompleted(rs.Status)
+}
+
+// runCompleted returns true if the run is completed
+func runCompleted(status string) bool {
+	// The following Run statuses indicate the completion
+	switch status {
+	case string(tfc.RunApplied):
+		return true
+	case string(tfc.RunPlannedAndFinished):
+		return true
+	case string(tfc.RunErrored):
+		return true
+	case string(tfc.RunCanceled):
+		return true
+	case string(tfc.RunDiscarded):
+		return true
+	}
+
+	return false
+}
+
+func (rs *RunStatus) RunApplied() bool {
+	return runCompleted(rs.Status)
```

Comment:
Was this supposed to be a call to `runApplied`?

Reply:
Yes, thanks! Great catch! Fixed.